SHARE-323 Materialize Form Controls

FelixAuf opened the pull request:
Changed textfields, textareas and radio buttons to material web components. Did not change them in the profile view because this view is going to be changed completely in SHARE-274 anyways.

---
### Your checklist for this pull request
Please review the [contributing guidelines](https://github.com/Catrobat/Catroweb-Symfony/blob/develop/.github/contributing.md) and [wiki pages](https://github.com/Catrobat/catroweb-Symfony/wiki/) of this repository.

- [x] Include the name and id of the Jira ticket in the PR’s title eg.: `SHARE-666 The devils ticket`
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no warnings and errors 
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Verify that all tests are passing (CI), if not please state the test cases in the [section](#Tests) below
- [x] Perform a self-review of the changes
- [x] Stick to the project’s git workflow (rebase and squash your commits)
- [x] Verify that your changes do not have any conflicts with the base branch
- [x] Put your ticket into the `Code Review` section in [Jira](https://jira.catrob.at/)
- [ ] Post a message in the *#catroweb* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer
- [x] Check that your pull request has been successfully deployed to https://web-test-1.catrob.at/

### Additional Description
`TODO: Add additional information that is not in your commit-message here`

### Tests - additional information
`TODO: add additional information about testruns here`

HCrane commented on templates/Program/program.html.twig:
```diff
@@ -137,7 +137,7 @@
 
   <script src="{{ asset('js/Program.min.js') }}"></script>
-  <script src="{{ asset('js/ProgramReport.min.js') }}"></script>
+  <script src="{{ asset('js/ProgramReport.min.js') }}" id="report"></script>
```
Can be removed as discussed

HCrane reviewed (approved):
From technical standpoint everything is in order. We are awaiting UI/UX response and PO response to merge it.

dmetzner commented on public/build/vendors~login~program_comments~program_description~register~report~request~reset.js:
```diff
@@ -0,0 +1,4195 @@
+(window["webpackJsonp"] = window["webpackJsonp"] || []).push([["vendors~login~program_comments~program_description~register~report~request~reset"],{
```
Those public/build/* files seem all autogenerated. Should they really be in the repository? or only generated locally?

FelixAuf replied:
Discussed with @HCrane already, for some reason if those files are not pushed, the javascript does not work (see https://github.com/Catrobat/Catroweb/pull/1070 where I did not push them). Tests work if only entrypoints.json is pushed, but then we can't see the behavior on web-test

dmetzner commented:
![image](https://user-images.githubusercontent.com/40868718/98537621-cbcce980-2289-11eb-937e-93c892cf5234.png)

The login page text fields look a bit broken (chrome)

FelixAuf commented:
> ![image](https://user-images.githubusercontent.com/40868718/98537621-cbcce980-2289-11eb-937e-93c892cf5234.png)
> 
> The login page text fields look a bit broken (chrome)

This seems to be a problem with chrome's autofill feature, found a workaround for the box surrounding the input (edit: does not seem to work on web-test for some reason), but cannot get the label to float if autofilled. If you enter anything with your keyboard this problem disappears

FelixAuf committed:
SHARE-323 Materialize Form Controls

Changed textfields, textareas, language selection and radio buttons to material web components. Did not change them in the profile view because this view is going to be changed completely in SHARE-274 anyways.
